README.md

```diff
@@ -14,6 +14,10 @@ Originally it was an ongoing project in 2015-16, but it is stagnated and now is
 
 ## Installation
 
+### Before compiling: Dependencies
+
+Since for testing, the `googltest` framework is used, the corresponding package has to be installed in order for the project to compile.
```

Comment:
You can also run `meson wrap install gtest` and commit the resulting file `subprojects/gtest.wrap` to git.

Then if googletest is already installed, it will be used. If not, instead of failing with an error, meson will simply resolve the `dependency()` call by using that little ini file to download the googletest source code and build it for the sake of linking to it in your testsuite.

Reply:
Thanks for the suggestion! This seems to do the trick